src/platform/packages/private/kbn-grid-layout/grid/grid_section/grid_section_header.tsx

```diff
@@ -174,6 +174,10 @@ export const GridSectionHeader = React.memo(({ sectionId }: GridSectionHeaderPro
 
     section.isCollapsed = !section.isCollapsed;
     gridLayoutStateManager.gridLayout$.next(newLayout);
+
+    const buttonRef = collapseButtonRef.current;
+    if (!buttonRef) return;
+    buttonRef.setAttribute('aria-expanded', `${!section.isCollapsed}`);
```

Comment:
This fixes a bug introduced in https://github.com/elastic/kibana/pull/218900 where I forgot to set `aria-expanded`